### PR TITLE
include custom assets for integration

### PR DESCRIPTION
**Pull request by karmab**

### CODE REVIEW

File: feature-configs/integration-kni1/dpdk/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/dpdk
```
Comment (tripledes) [inline feature-configs/integration-kni1/dpdk/kustomization.yaml]: The path should be `../../demo/dpdk`

File: feature-configs/integration-kni1/performance/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/performance
```
Comment (tripledes) [inline feature-configs/integration-kni1/performance/kustomization.yaml]: This one, `../../base/performance`

File: feature-configs/integration-kni1/ptp/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/ptp
```
Comment (tripledes) [inline feature-configs/integration-kni1/ptp/kustomization.yaml]: `../../base/ptp`

File: feature-configs/integration-kni1/sctp/kustomization.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/sctp
```
Comment (tripledes) [inline feature-configs/integration-kni1/sctp/kustomization.yaml]: `../../base/sctp`

File: feature-configs/integration-kni1/sriov/kustomization.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/sriov
```
Comment (tripledes) [inline feature-configs/integration-kni1/sriov/kustomization.yaml]: `../../base/sriov`

File: feature-configs/integration-u08/dpdk/kustomization.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/dpdk
```
Comment (tripledes) [inline feature-configs/integration-u08/dpdk/kustomization.yaml]: `../../demo/dpdk`

File: feature-configs/integration-u08/performance/kustomization.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/performance
```
Comment (tripledes) [inline feature-configs/integration-u08/performance/kustomization.yaml]: `../../base/performance`

File: feature-configs/integration-u08/ptp/kustomization.yaml [hunk 8]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/ptp
```
Comment (tripledes) [inline feature-configs/integration-u08/ptp/kustomization.yaml]: `../../base/ptp`

File: feature-configs/integration-u08/sctp/kustomization.yaml [hunk 9]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/sctp
```
Comment (tripledes) [inline feature-configs/integration-u08/sctp/kustomization.yaml]: `../../base/sctp`

File: feature-configs/integration-u08/sriov/kustomization.yaml [hunk 10]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/sriov
```
Comment (tripledes) [inline feature-configs/integration-u08/sriov/kustomization.yaml]: `../../base/sriov`

File: feature-configs/integration-vms/dpdk/kustomization.yaml [hunk 11]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/dpdk
```
Comment (tripledes) [inline feature-configs/integration-vms/dpdk/kustomization.yaml]: `../../demo/dpdk`

File: feature-configs/integration-vms/performance/kustomization.yaml [hunk 12]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/performance
```
Comment (tripledes) [inline feature-configs/integration-vms/performance/kustomization.yaml]: `../../base/performance`

File: feature-configs/integration-vms/ptp/kustomization.yaml [hunk 13]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/ptp
```
Comment (tripledes) [inline feature-configs/integration-vms/ptp/kustomization.yaml]: `../../base/ptp`

File: feature-configs/integration-vms/sctp/kustomization.yaml [hunk 14]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/sctp
```
Comment (tripledes) [inline feature-configs/integration-vms/sctp/kustomization.yaml]: `../../base/sctp`

File: feature-configs/integration-vms/sriov/kustomization.yaml [hunk 15]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/sriov
```
Comment (tripledes) [inline feature-configs/integration-vms/sriov/kustomization.yaml]: `../../base/sriov`

File: feature-configs/integration-wl4-d17u01b05/performance/kustomization.yaml [hunk 16]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../bases/performance
```
Comment (tripledes) [inline feature-configs/integration-wl4-d17u01b05/performance/kustomization.yaml]: `../../base/performance`

File: feature-configs/integration-vms/performance/performance_profile_patch.yaml [hunk 17]
```diff
@@ -0,0 +1,18 @@
+apiVersion: performance.openshift.io/v1alpha1
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker-cnf: ""
+  cpu:
+    isolated: "6-11"
+    nonIsolated: "0-5"
```
Comment (slintes) [inline feature-configs/integration-vms/performance/performance_profile_patch.yaml]: we don't have nonIsolated anymore

File: feature-configs/integration-wl4-d17u01b05/performance/performance_profile_patch.yaml [hunk 18]
```diff
@@ -0,0 +1,18 @@
...
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker-cnf: ""
+  cpu:
+    isolated: "8-15,24-31"
+    reserved: "0-7,16-23"
+    nonIsolated: "0-7,16-23"
```
Comment (slintes) [inline feature-configs/integration-wl4-d17u01b05/performance/performance_profile_patch.yaml]: we don't have nonIsolated anymore

**Pull request merged**